@types/foundry-pc-types/types/game.d.ts

```diff
@@ -0,0 +1,213 @@
+// @TODO: Add types
+
+declare let socket: SocketIOClient.Socket;
+declare let canvas: any;
+declare let keyboard: any;
+declare let game: Game;
+declare let ui: {
+	notifications: Notifications;
+	tables: RollTableDirectory;
+	combat: CombatTracker;
+	actors: ActorDirectory;
+	windows: Record<number, Application>;
+};
+
+declare const _templateCache: Record<string, unknown>;
+
+/**
+ * The core Game instance which encapsulates the data, settings, and states relevant for managing the game experience.
+ * The singleton instance of the Game class is available as the global variable ``game``.
+ *
+ * @param worldData	An object of all the World data vended by the server when the client first connects
+ * @param userId	The ID of the currently active user, retrieved from their session cookie
+ * @param socket	The open web-socket which should be used to transact game-state data
+ */
+declare class Game {
+	/** The object of world data passed from the server */
+	data: any;
+
+	/** Localization support */
+	i18n: Localization;
+
+	/** The Keyboard Manager */
+	keyboard: KeyboardManager;
+
+	/** A mapping of installed modules */
+	modules: Map<any, any>;
+
+	/** The user role permissions setting */
+	permissions: any;
+
+	/** The client session id which is currently active */
+	sessionId: string;
+
+	/** Client settings which are used to configure application behavior */
+	settings: ClientSettings;
+
+	/** A reference to the open Socket.io connection */
+	socket: SocketIOClient.Socket;
+
+	/** The id of the active game user */
+	userId: string;
+
+	/** A singleton instance of the Audio Helper class */
+	audio: AudioHelper;
+
+	/** A singleton instance of the Video Helper class */
+	video: VideoHelper;
+
+	/** Whether the Game is running in debug mode */
+	debug: boolean;
+
+	/**
+	 * A flag for whether texture assets for the game canvas are currently loading
+	 */
+	loading: boolean;
+
+	/** A flag for whether the Game has successfully reached the "ready" hook */
+	ready: boolean;
+
+	/* -------------------------------------------- */
+	/*  Entities
+	/* -------------------------------------------- */
+
+	users: Users;
+	messages: Messages;
+	scenes: Scenes;
+	actors: Actors;
+	items: Items;
+	journal: Journal;
+	macros: Macros;
+	playlists: Playlists;
+	combats: CombatEncounters;
```

Comment:
Codacy has a fix for the issue: [Expected blank line between class members.](https://app.codacy.com/gh/bmarian/token-tooltip-alt/pullRequest?prid=7194732)
```suggestion

```